pkgs/firehose/bin/firehose.dart

```diff
@@ -20,10 +20,11 @@ void main(List<String> arguments) async {
 
     var validate = argResults['validate'] == true;
     var publish = argResults['publish'] == true;
+    var health = argResults['health'] != null;
```

Comment:
This lgtm as is; we could also (now or in a future PR) refactor so that `bin/health.dart` is a new entry-point; it's separate enough from the publishing work that it's worth splitting it out.